kalpana.py

```diff
@@ -682,7 +681,7 @@ def signedArea(ring):
                     if filetype in timeVaryingFiles:
                         var = nc[fileTypesNetCDFVarNames[filetype]][i][:]
                     # construct mask
-                    if filetype in requireFill and var.mask.any():

```

Comment:
I don't get a masked array, even though my variable had _FillValue = -99999.
So I added a check for "MaskedArray" type.  Not sure if this is the way to deal with it. There may be a cleaner way to address this, like making sure one enters with a MaskedArray.